test/unit/cleanup-tests.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/helper.sh
+
+readonly _CLEANUP_SCRIPT="tools/cleanup/cleanup.sh"
+source "$(dirname ${BASH_SOURCE})/../../${_CLEANUP_SCRIPT}"
```

Comment:
`dirname $0` is cleaner, and this script is not intended to be sourced.

```suggestion
source $(dirname $0)/../../tools/cleanup/cleanup.sh
```